index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portifolio</title>
+    <link rel="stylesheet" href="./assets/css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap" rel="stylesheet">
+</head>
+<body>
+   <header>
+       <nav class="navbar">
+           <div class="menubar">
+            <p>North Code</p>
+            <img src="./assets/images/bars/IconMenu.png" alt="menuicon">
+           </div>
+       </nav>
+   </header>
+
+<section class="headline">
+  <div class="headtext">
+      <h1 class="primary-text"> Kinginthenorth codez</h1>
+      <h3 class="sub-text"> Software Developer</h3>
+  </div>
+  <div class="surport-text">
+      <div class="intro">
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded,  
+            don’t hestiate to contact me.
+          </p>
+      </div>
+      <hr class="divider">
+      <div class="items">
+          <div class="item">
+            <h3>Language</h3>
+            <ul class="list-items">
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>Css</li>
+            </ul>
+          </div>
+          <div class="item">
+            <h3>Frameworks</h3>
+            <ul class="list-items">
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
+            </ul>
+          </div>
+        <div class="item">
+          <h3>Skills</h3>
+          <ul class="list-items">
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Termial</li>
+          </ul>
+        </div>
+      </div>
+      <hr class="divider">
+      <div class="social">
+       <img src="./assets/images/GitHub.svg" alt="icon">
+       <img src="./assets/images/Angelist.svg" alt="icon">
+       <img src="./assets/images/Linkedin.svg" alt="icon">
+       <img src="./assets/images/Twitter.svg" alt="icon">
+       <img src="./assets/images/Medium.svg" alt="icon">
+      </div>
+      <div class="more">
+        <img src="./assets/images/Group55.svg" alt="icon">
+        <p>Work</p>
+      </div>  
+  </div>
```

Comment:
- [ OPTIONAL] take a look at the red line I left in, I mentioned them as optional because the previous has forgotten to talk about it, but  I strongly recommend you make these changes as you move forward into the next projects

- YOUR DESIGN

<img width="315" alt="king" src="https://user-images.githubusercontent.com/79058364/158894768-1a8eb0d7-fb1c-417f-9622-1bbec33c0f60.png">


- [ FIGMA] 

<img width="248" alt="violet" src="https://user-images.githubusercontent.com/79058364/158894840-22662639-6e0c-40e0-8930-daf99d8f4e78.png">